docs/pages/access-controls/access-request-plugins/ssh-approval-pagerduty.mdx

```diff
@@ -9,27 +9,13 @@ that can become a vector for attacks.
 
 Teleport's PagerDuty integration allows you to treat Teleport Role Access
 Requests as PagerDuty incidents, notify the appropriate on-call team, and
-approve or deny the requests via PagerDuty. You can also configure the plugin to
+approve or deny the requests via Teleport. You can also configure the plugin to
 approve Role Access Requests automatically if the user making the request is on
 the on-call team for a service affected by an incident.
 
 This guide will explain how to set up Teleport's Access Request plugin for
 PagerDuty.
 
-<video controls>
-  <source
-    src="../../../img/enterprise/plugins/pagerduty/pagerduty-demo.mp4"
```

Comment:
Should we actually remove the files from the repo too? (Also I can hardly believe we actually have mp4 files in git. 😅)

Reply:
Removed